src/phases/execution/RunLargeArithmeticOp.yml

```diff
@@ -0,0 +1,25 @@
+
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This phase template constructs an aggregation pipeline that multiplies together
+  the provided arguments.
+
+RunMultiply:
+  Repeat: 10
+  Database: {^Parameter: {Name: "Database", Default: "test"}}
+  Operations:
+  - OperationMetricsName: {^Parameter: {Name: "Name", Default: "RunMultiply"}}
+    OperationName: RunCommand
+    OperationCommand:
+      aggregate: {^Parameter: {Name: "Collection", Default: "Collection0"}}
+      pipeline: [
+        {
+          $group: {
+            _id: {
+              $multiply: {^Parameter: {Name: "Expression", Default: []}}
```

Comment:
Curious why the $group? Would it be sufficient to project a new field with the computation?

Reply:
This matches the BFs that were reported by the fuzzer -- right now $project is pushed down fully to SBE and that might be a reason there are no fuzzer BFs with $project. One thing I'm thinking about is if it makes sense to run this workload on a variant that forces the classic engine. One of my todos still is reaching out to QE to see how SBE handles this and if it makes sense to be testing SBE too.